docs/kube-up.md

```diff
@@ -0,0 +1,22 @@
+# Production Quality Cluster on GCE
+This document provides the steps to bring up a production quality cluster on GCE with [`kube-up.sh`](https://kubernetes.io/docs/getting-started-guides/gce/).
+
+## Download CRI-Containerd Release Tarball
+To download release tarball, see [step 1](./installation.md#step-1-download-cri-containerd-release-tarball) in installation.md.
+
+Unpack release tarball to any directory, using `${CRI_CONTAINERD_PATH}` to indicate the directory in the doc:
+```shell
+tar -C ${CRI_CONTAINERD_PATH} -xzf cri-containerd-${VERSION}.linux-amd64.tar.gz
+```
+## Set Environment Variables for CRI-Containerd
+Use environment variable `CRI_CONTAINERD_VERSION` to specify `cri-containerd` version. By default,
+latest version will be used.
+```shell
+. ${CRI_CONTAINERD_PATH}/cluster/gce/env
```

Comment:
$

Reply:
Just by looking at it I am not sure what ``` ${CRI_CONTAINERD_PATH}/cluster/gce/env``` is doing . Does it setup the gce env ?

Reply:
Yeah, it setup the environment variables useful to `kube-up.sh`.

Reply:
Done